ee/modules/600-flant-integration/images/madison-proxy/madison-proxy.go

```diff
@@ -0,0 +1,122 @@
+/*
+Copyright 2021 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package main
+
+import (
+	"context"
+	"crypto/tls"
+	"net/http"
+	"net/http/httputil"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const logFormat = `request:%s %s %s, status:%s, body:%s`
+
+// This type implements the http.RoundTripper interface
+type LoggingRoundTripper struct {
+	Proxied http.RoundTripper
+}
+
+func (lrt LoggingRoundTripper) RoundTrip(req *http.Request) (res *http.Response, e error) {
+	dump, err := httputil.DumpRequestOut(req, true)
+	if err != nil {
+		log.Error(err)
+	}
+	// Send the request, get the response (or the error)
+	res, e = lrt.Proxied.RoundTrip(req)
+
+	log.Infof("%s", string(dump))
+	if res != nil {
+		log.Infof("response: %s", res.Status)
+	}
+	return
+}
+
+var (
```

Comment:
This would be better to configure these options with environment variables, but global variables are also ok for our case anyway.

Reply:
yep

Reply:
it can be set from envs, but have default values now